pkg/services/sqlstore/migrations/alert_mig.go

```diff
@@ -82,4 +82,18 @@ func addAlertMigrations(mg *Migrator) {
 		{Name: "type", Type: DB_NVarchar, Length: 255, Nullable: false},
 		{Name: "settings", Type: DB_Text, Nullable: false},
 	}))
+	active_node := Table{
+		Name: "active_node",
+		Columns: []*Column{
+			{Name: "id", Type: DB_BigInt, IsPrimaryKey: true, IsAutoIncrement: true},
+			{Name: "node_id", Type: DB_NVarchar, Length: 255, Nullable: false},
+			{Name: "heartbeat", Type: DB_TimeStampz, Nullable: false},
+			{Name: "sequence", Type: DB_Int, Nullable: false},
+		},
+		Indices: []*Index{
+			{Cols: []string{"node_id", "heartbeat"}, Type: UniqueIndex},
```

Comment:
Heartbeat are to be normalized to the nearest minute. Refer to examples at https://confluence.walmart.com/display/STRMALERT/Clustering+of+Alert+Processor+using+DB+approach#ClusteringofAlertProcessorusingDBapproach-DetectingMissingAlerts

Also, we need not pass in timestamp and use db timestamp (i.e. similar to now() in postgres etc.). Refer to comments are API#2 - https://confluence.walmart.com/display/STRMALERT/Clustering+of+Alert+Processor+using+DB+approach#ClusteringofAlertProcessorusingDBapproach-APIs